targets/unity-v2/source/ExampleTestProject/Assets/PlayFabEditorExtensions/Editor/Scripts/Panels/PlayFabEditorAuthenticate.cs

```diff
@@ -301,6 +308,76 @@ private static void OnLoginButtonClicked()
             });
         }
 
+        private static async void OnAADLoginButtonClicked()
+        {
+            string[] scopes = new string[] { string.Format("{0}/.default", PlayFabEditorHelper.ED_EX_AAD_SIGNIN_CLIENTID) };
```

Comment:
I've added a new scope `plugin` and pre-authorized it. `.default` = `plugin`

Reply:
Thanks! Also, it looks like we need to shift the app id associated with that scope in this code. Instead of the client app id, we should use the service app id. So, the full scope would be: "448adbda-b8d8-4f33-a1b0-ac58cf44d4c1/plugin"